lib/ui/text.dart

```diff
@@ -360,17 +360,17 @@ class TextDecoration {
   const TextDecoration._(this._mask);
 
   /// Creates a decoration that paints the union of all the given decorations.
-  factory TextDecoration.combine(List<TextDecoration> decorations) {
+  factory TextDecoration.combine(List<TextDecoration/*!*/>/*!*/ decorations) {
```

Comment:
Unrelated question: How does nnbd handle factory constructors that return null?

Reply:
Factories are not allowed to return null any more.